docs/source/user_guide/runtime-conf.md

```diff
@@ -122,6 +122,24 @@ elyra-metadata install runtimes \
 
 Refer to the [Kubeflow Pipelines Configuration settings](#kubeflow-pipelines-configuration-settings) section for an explanation of the parameters. 
 
+#### Exporting runtime configurations
+
+To export runtime configurations:
+
+```bash
+elyra-metadata export runtimes \
+	--schema-name="kfp" \
```

Comment:
```suggestion
	--schema_name="kfp" \
```

Reply:
Removed all mentions of `--schema-name` parameter from the documentation as discussed via chat.